Hercules-DJControl-Inpulse-200-script.js

```diff
@@ -0,0 +1,249 @@
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global print                                                       */
+/* global midi                                                        */
+//////////////////////////////////////////////////////////////////////// 

```

Comment:
Please remove the JSHint config, this conflicts with eslint that we're using now.